src/main/java/org/jabref/logic/importer/fileformat/MrDLibImporter.java

```diff
@@ -255,7 +255,4 @@ public void characters(char ch[], int start, int length) throws SAXException {
         }
 
     }
-
-    ;
```

Comment:
It's really weird what kind of code you still find in JabRef. Great job removing this!